tests/testthat/test-get_table.R

```diff
@@ -74,14 +74,15 @@ test_that("get_table returns list of tables if no table is requested", { for (co
 test_that("table_exists fails when multiple matches are found", {
   for (conn in conns) {
     # This test exploits ambiguous notation requiring schemas to exits
-    if (!inherits(conn, "PqConnection")) next
+    if (!inherits(conn, "SQLiteConnection")) next
```

Comment:
You dont want this test to run on Postgres?

Reply:
The tests exploit the same case, so I would actually be in favor of keeping the test only on SQLite, which would in turn no longer require the schema `test.one` to exist in the Postgres database.

Reply:
Additionally, as the test now calls [`ATTACH [ expr ] AS [ schema-name ]`](https://www.sqlite.org/lang_attach.html) &mdash; which is most likely SQLite only &mdash; I decided against writing a "universal" test, as any difference in results across backends would trace back to `get_tables` behaving inconsistently across backends.

Also, as this test is the only one which requires a specific schema ("test.one") to exist, it's better to quickly attach a temporary file in SQLite than to write a test for all supported backends.

For once, SQLite reigns supreme.